1-js/02-first-steps/05-types/article.md

```diff
@@ -178,7 +179,7 @@ O código acima declara que `age` é desconhecido.
 
 O valor especial `undefined` também se diferencia. Faz um tipo próprio, tal como `null`.
 
-O significado de `undefined` é "o valor não é atribuído".
+O significado de `undefined` é "valor não é atribuído".
```

Comment:
The meaning of `undefined` is **indefinido**, and when we talk about data types and values, it means **valor indefinido**.

To consider it as *valor não atribuído*, it should set as `null` that means absence of value. `undefined` means that the value and its type are not defined.

Reply:
It was maintained according to the original excerpt from the repository in English and when the initial translation occurred @nazarepiedady, see content in English https://github.com/javascript-tutorial/en.javascript.info/blob/master/1-js/02-first-steps/05-types/article.md#the-undefined-value

![image](https://github.com/javascript-tutorial/pt.javascript.info/assets/35238611/20d21490-2ab8-40c2-852d-9b77d341da26)

But I also consider the adjustment valid, being direct to say that it means `indefinido` makes more sense, although we would change the original content in English

Reply:
Taking the things that way you are right, so let's keep the original approach of defining it.